mix.exs

```diff
@@ -66,7 +66,8 @@ defmodule PhxTools.MixProject do
       {:dialyxir, "~> 1.2", only: [:dev, :test], runtime: false},
       {:excoveralls, "~> 0.15", only: [:dev, :test]},
       {:mix_audit, "~> 2.1", only: [:dev, :test], runtime: false},
-      {:sobelow, "~> 0.8", only: [:dev, :test], runtime: false}
+      {:sobelow, "~> 0.8", only: [:dev, :test], runtime: false},
+      {:ua_parser, "~> 1.8"}
```

Comment:
Does the library require any additional steps for setup? Add those to the README.md.

Reply:
Library does not require any additional steps.